README.md

```diff
@@ -36,6 +45,27 @@ Return to your original locations.  In this step all groups will act like Design
 
 ***Q1.3 Briefly describe your solutions (use a list) and any strengths/weaknesses you see.***
 
+1. smart washing machines
+  * Strengths
+    * configurable
+    * connected
+    * user-friendly (simple)  
+    * commonplace/widespread
+  * Weaknesses
+    * hackable
+    * cost? (need to purchase washing machine)
```

Comment:
In terms of cost, an ideal situation that looks more towards the future would involve washing machines that are made to be compatible with the mobile application you mentioned above. This would require communication between the companies behind the machines and the people working on the app.